docs/csharp/index.md

```diff
@@ -23,7 +23,7 @@ The C# guide provides a wealth of information about the C# language. This site h
     - If you've done development before, but are new to C#, read the [tour of the C# language](tour-of-csharp/index.md). You will learn the basic syntax and structure for the language, and you can use the language tour to contrast C# with other languages you've used. You can also browse the [tutorials](tutorials/index.md) to try basic C# programs. Or, check out the [C# Concepts](concepts.md) to learn the features of the C# language in depth.
 
 * Experienced C# developers:
-    - If you've used C# before, you should start by reading what's in the latest version of the language. Check out [What's new in C# 6](csharp-6.md) for the new features in the current version. Then, explore the [C# Concepts](concepts.md) where you want more depth.
+    - If you've used C# before, you should start by reading what's in the latest version of the language. Check out [What's new in C# 6](whats-new/csharp-6.md) for the new features in the current version. Then, explore the [C# Concepts](concepts.md) where you want more depth.
```

Comment:
Since we're updating this now, this should probably now direct readers to the new in C# 7 document instead of C# 6.
Or, we could make it evergreen, and direct people to the overall "what's new" page.